main.nf

```diff
@@ -1975,7 +1975,7 @@ process manta_panel {
 	stageOutMode 'copy'
 
 	when:
-		params.sv && params.onco
+		params.sv && params.antype == "panel"
```

Comment:
Here exome and myeloid are again included in addition to modycf

Reply:
that's not too bad though, since myeloid has params.sv = false. But one could make all these params much smarter instead of using not wgs and antype = panel and so on.

Reply:
Yes, I am a bit intimidated by all these conditions at the moment. Me or someone else will for sure mess things up when we do further changes here in the future. If you can think of a way to present it in a "smarter" way, that would be helpful.